ui/src/app/components/plans/details-header.component.html

```diff
@@ -123,12 +123,12 @@
       <a
         [routerLink]="['devices']"
         [routerLinkActive]="'active'"
-        class="normal-text">
+        class="normal-text d-flex align-items-center">
         <i class="fz-15"
           [class.fa-desktop]="!version?.workspace.isMobile"
           [class.fa-mobile-alt-solid]="version?.workspace.isMobile"></i>
         <span [translate]="'test_plans.test_devices.title'" *ngIf="version?.workspace.isMobile"></span>
-        <span [translate]="'test_plans.test_machines.title'" *ngIf="!version?.workspace.isMobile"></span>
+        <span style="padding-bottom: 1px" [translate]="'test_plans.test_machines.title'" *ngIf="!version?.workspace.isMobile"></span>
```

Comment:
can you plz use class name and reduce inline style usage?

Reply:
done bro